onnxruntime/core/common/cpuid_info.cc

```diff
@@ -57,10 +70,10 @@ CPUIDInfo CPUIDInfo::instance_;
 
 CPUIDInfo::CPUIDInfo() {
 #if (defined(CPUIDINFO_ARCH_X86) || defined(CPUIDINFO_ARCH_ARM)) && defined(CPUINFO_SUPPORTED)
-    if (!cpuinfo_initialize()) {
-      // Unfortunately we can not capture cpuinfo log!!
-      ORT_THROW("Failed to initialize CPU info.");
-    }
+  pytorch_cpuinfo_init_ = cpuinfo_initialize();
```

Comment:
nit: move this to CPUIDINFO_ARCH_ARM section.

Reply:
hope is that cpuinfo should be used under more platforms , not just arm